src/foam/core/CompoundException.js

```diff
@@ -68,6 +62,11 @@ foam.CLASS({
   ],
 
   methods:  [
+    {
+      name: 'getExceptions',
```

Comment:
Do we need this? Could we just use getSuppressed directly?

Reply:
Thought we could preserve the concept of a compound exception having a list exception and the client code could just stay the same `compoundException.getException()`. IMO, getSuppressed() is quite java sided.